### PR TITLE
Updated Server.md doc

### PR DESCRIPTION
**Pull request by vrakesh**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/server.md [hunk 1]
```diff
@@ -79,133 +78,73 @@ After this deep dive, you might also be interested in:
...
 ```bash
 $ mxnet-model-server --help
-usage: mxnet-model-server [-h] --models KEY1=VAL1 KEY2=VAL2...
-                          [KEY1=VAL1 KEY2=VAL2... ...] [--service SERVICE]
-                          [--gen-api GEN_API] [--port PORT] [--host HOST]
-                          [--gpu GPU] [--log-file LOG_FILE]
-                          [--log-rotation-time LOG_ROTATION_TIME]
-                          [--log-level LOG_LEVEL]
-                          [--metrics-write-to {log,csv,cloudwatch}]
+usage: mxnet-model-server [-h] (--start | --stop) [--mms-config MMS_CONFIG]
```
Comment (vdantu) [inline docs/server.md]: start and stop are optional. You might need to change the brackets from () to []

File: docs/server.md [hunk 2]
```diff
@@ -9,18 +9,18 @@
 
 ## Overview
 
-The primary feature of MMS is the model server. It can be used for many types of inference in production settings. It provides an easy-to-use command line interface and utilizes an industry standard [OpenAPI interface](rest_api.md). It has features for pre-processing and post-processing inputs and outputs for customized applications. Support for models from a wide range of deep learning frameworks is achieved through its [ONNX model](https://onnx.ai) export feature.
+MMS model serving can be used for many types of inference in production settings. It provides an easy-to-use command line interface and utilizes a [RESTful API](rest_api.md) handle state prediction requests. Support for models from a wide range of deep learning frameworks is achieved through its [ONNX model](https://onnx.ai) export feature.
```
Comment (vdantu) [inline docs/server.md]: Expand MMS to "MXNet Model Server" ... We wouldn't need model serving again.

Comment (vrakesh) [reply docs/server.md]: ah I agree

File: docs/server.md [hunk 3]
```diff
@@ -30,45 +30,44 @@ To test it out, you will need to open a new terminal window next to the one runn
...
-      {
-        "class": "n02123394 Persian cat",
-        "probability": 0.00026897044153884053
-      }
-    ]
-  ]
-}
-```
-You will see this result in the response to your `curl` call to the predict endpoint, and depending on the logging options selected, in the terminal window running MMS, and/or log files.
+You will see this result in the response to your `curl` call to the predict endpoint, in the terminal window running MMS and log files.
```
Comment (vdantu) [inline docs/server.md]: nitpick, predictions endpoint

File: docs/server.md [hunk 4]
```diff
@@ -30,45 +30,44 @@ To test it out, you will need to open a new terminal window next to the one runn
...
 
 After this deep dive, you might also be interested in:
-* [Metrics](metrics.md): what kind of metrics and logs are available
+* [Logging](logging.md): what kind of logs are available
```
Comment (vdantu) [inline docs/server.md]: Is this a question? If so, you might want to end the sentence with ? :D

File: docs/server.md [hunk 5]
```diff
@@ -79,162 +78,101 @@ After this deep dive, you might also be interested in:
...
-This would serve a prediction endpoint at `/not-hot-dog/predict` and run your custom service code that is located in `nothotdog.py`. For more info on custom services, check out the [object detection example](../examples/ssd/README.md) and the [custom service documentation](custom_service.md).
+This would serve a prediction endpoint at `predictions/not-hot-dog/` and run your custom service code in the archive, the manifest in archive would point to the entry point.
 
 ### Serving Multiple Models with MMS
 
 Example multiple model usage:
 
 ```bash
-mxnet-model-server --models name=model_location name2=model_location2
+mxnet-model-server --start --models name=model_location name2=model_location2
```
Comment (vdantu) [inline docs/server.md]: You can remove --start

File: docs/server.md [hunk 6]
```diff
@@ -79,162 +78,101 @@ After this deep dive, you might also be interested in:
...
 ```
 
 Here's an example for running the resnet-18 and the vgg16 models using local model files.
 
 ```bash
-mxnet-model-server --models resnet-18=file://models/resnet-18 squeezenet=file://models/squeezenet_v1.1
+mxnet-model-server --start --models resnet-18=file://models/resnet-18.mar squeezenet=file://models/squeezenet_v1.1.mar
```
Comment (vdantu) [inline docs/server.md]: Remove --start ... Its optional, and removing it keeps it in sync with mms 0.4

**Pull request merged**